### PR TITLE
build: autodetect and install correct wasm-bindgen-cli version

### PR DESCRIPTION
**Pull request by lklimek**
## Issue being fixed or feature implemented

wasm-bindgen-cli version must match wam-bindgen dependency version

## What was done?

* install cargo-lock in deps image
* autodetect wasm-bindgen-cli version in build-wasm.sh
* bump wasm-bindgen to 0.2.85

## How Has This Been Tested?

Local network, Github Actions

## Breaking Changes

Locally installed version of wasm-bindgen-cli will be overwritten.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

**Review by strophy — DISMISSED** [review]

File: .github/actions/docker/action.yaml [hunk 1]
```diff
@@ -9,7 +9,13 @@ inputs:
...
     required: true
   target:
-    description: "Target stage within Dockerfile: dapi, drive-abci, dashmate-helper, test-suite"
+    description: Target stage within Dockerfile
+    type: choice
+    options:
+      - dapi
+      - drive-abci
+      - dashmate-helper
+      - test-suite
```
Comment (shumkov) [inline .github/actions/docker/action.yaml:18]: These params are not supported for action input, they are only for workflow input which is different

**Review by shumkov — CHANGES_REQUESTED** [review]
Multiple tests are failed

File: Dockerfile [hunk 2]
```diff
@@ -114,14 +114,19 @@ ENV NODE_ENV ${NODE_ENV}
 # Install wasm-bindgen-cli in the same profile as other components, to sacrifice some performance & disk space to gain
 # better build caching
 WORKDIR /platform
+# TODO: refactor to not need the wasm-bindgen-cli and remove the copy below, as deps stage should be independent
+COPY Cargo.lock .
```
Comment (shumkov) [inline Dockerfile:118]: that will invalidate layer cache pretty often. maybe we just stick on specific version of this library?

**Review by shumkov — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (shumkov) [reply Dockerfile:118]: Agreed to revert changes in Dockerfile

File: Dockerfile [hunk 3]
```diff
@@ -121,7 +121,9 @@ RUN --mount=type=cache,sharing=shared,target=/root/.cache/sccache \
     --mount=type=cache,sharing=private,id=target_${TARGETARCH},target=/platform/target \
     export SCCACHE_SERVER_PORT=$((RANDOM+1025)) && \
     if [[ -z "${SCCACHE_MEMCACHED}" ]] ; then unset SCCACHE_MEMCACHED ; fi ; \
-    CARGO_TARGET_DIR=/platform/target cargo install --profile "$CARGO_BUILD_PROFILE" wasm-bindgen-cli@0.2.84
+    export CARGO_TARGET_DIR=/platform/target ; \
+    cargo install --profile "${CARGO_BUILD_PROFILE}" wasm-bindgen-cli@0.2.85 && \
```
Comment (shumkov) [inline Dockerfile:125]: Do we actually need to use profile here?

**Review by shumkov — APPROVED** [review]
👍

**Pull request merged**